docs/guides/self-hosted/quick-start.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: Quick start
+description: Learn how to self-host Latitude
+---
+
+TODO.
```

Comment:
let's remove this page for now unless we fill it with content

Reply:
The issue though is that there is currently a link to this page from the main quick start guide, which confusingly redirects back to the getting started introduction.
<img width="701" alt="image" src="https://github.com/user-attachments/assets/6cb58ea8-fe0e-4afd-be59-ec96addf3e53">

Reply:
lol ok let's move forward then cc @samulatitude this probably needs reviewing at some point